static/cities/sydney_454084/locales/ru-RU.json

```diff
@@ -1,45 +1,3 @@
 {
-  "articlesSectionCtaButtonText": null,
-  "articlesSectionCtaDescription": null,
-  "articlesSectionSubtitle": null,
-  "articlesSectionTitle": null,
-  "itineraries.439277.tips.439276.description": null,
-  "itineraries.439277.tips.439276.title": null,
-  "itineraries.439277.tips.439278.description": null,
-  "itineraries.439277.tips.439278.title": null,
-  "itineraries.439277.tips.439279.description": null,
-  "itineraries.439277.tips.439279.title": null,
-  "itineraries.439277.tips.439280.description": null,
-  "itineraries.439277.tips.439280.title": null,
-  "itineraries.439277.tips.439281.description": null,
-  "itineraries.439277.tips.439281.title": null,
-  "itineraries.439277.tips.439282.description": null,
-  "itineraries.439277.tips.439282.title": null,
-  "itineraries.439277.tips.439283.description": null,
-  "itineraries.439277.tips.439283.title": null,
-  "itineraries.439277.title": null,
-  "itineraryCtaButtonText": "Начни свое путешествие",
-  "itinerarySectionSubtitle": null,
-  "itinerarySectionTitle": null,
-  "itinerarySelectorDescription": null,
-  "mainActionButtonText": null,
-  "mainDescription": null,
-  "mainTitle": null,
-  "metaDescription": null,
-  "metaTitle": null,
-  "name": null,
-  "otherMetaTags.454107.value": null,
-  "partnersSectionDescription": null,
-  "partnersSectionSubtitle": null,
-  "partnersSectionTitle": null,
-  "searchSectionSubtitle": null,
-  "searchSectionTitle": null,
-  "sliderSectionButtonText": null,
-  "sliderSectionDescription": null,
-  "sliderSectionLinkText": null,
-  "sliderSectionTitle": null,
-  "stickyBannerButtonText": null,
-  "stickyBannerText": null,
-  "videoSectionSubtitle": null,
-  "videoSectionTitle": null
+  "name": "Сидней"
```

Comment:
Is it intended that all keys were deleted except one?

Reply:
they are not deleted. we just should not fetch null values because otherwise all phraseApp locales will be fetched for each cities but we need to fetch only locales that are translated